src/css/betterttv-blue-buttons.css

```diff
@@ -212,3 +212,21 @@ div#header_actions a.primary span {
 .recent-past-broadcast {
     border-left: 5px solid #2F58B9;
 }
+
+.warp__item--toggled>a {
```

Comment:
you should scope things to pages and place them below a new comment block when it's a new page